bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -813,49 +682,48 @@ class PlotCanvas extends LayoutDOM.Model
     return constraints
 
   _get_constant_constraints: () ->
-    constraints = []
 
-    # Add the constraints that always apply for a plot
-    min_border_top    = @get('min_border_top')
-    min_border_bottom = @get('min_border_bottom')
-    min_border_left   = @get('min_border_left')
-    min_border_right  = @get('min_border_right')
-    frame             = @get('frame')
-    canvas            = @get('canvas')
+    min_border_top    = @plot.min_border_top
```

Comment:
I would have thought these constraints now belong on Plot given that you're dealing with min_border stuff there.